app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :post
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
Like model has a method that updates the likes counter for a post 😉 Nicely done!

Reply:
Great review @mateo951 ...thank you.